hack/lib/test.sh

```diff
@@ -394,16 +396,16 @@ kube::test::version::diff_assert() {
         echo "${bold}${red}"
         echo "FAIL! ${diff_msg}"
         echo "  Expected: "
-        echo "$(cat ${original})"
+        cat "${original}"
         echo "  Got: "
-        echo "$(cat ${latest})"
+        cat "${latest}"
         echo "${reset}${red}"
         caller
         echo "${reset}"
         return 1
     fi
   else
-    if [ ! -z "$(diff -iwB ${original}.sorted ${latest}.sorted)" ] ; then
+    if [ -n "$(diff -iwB "${original}".sorted "${latest}".sorted)" ] ; then
```

Comment:
Fix for this:

```
In ./hack/lib/test.sh line 406:
    if [ ! -z "$(diff -iwB ${original}.sorted ${latest}.sorted)" ] ; then
         ^-- SC2236: Use -n instead of ! -z.
                           ^---------^ SC2086: Double quote to prevent globbing and word splitting.
                                              ^-------^ SC2086: Double quote to prevent globbing and word splitting.
```